.trivyignore

```diff
@@ -1,2 +1,4 @@
 # According to govulncheck we are not using code that is affected by CVEs
+CVE-2025-22869
```

Comment:
@Prajyot-Parab, for future reference, can you please add description to PR on why these CVEs are ignored?

Reply:
Done